sfr/subchannel/THORS/FFM-5B.i

```diff
@@ -0,0 +1,255 @@
+# J. T. Han 1977,
```

Comment:
This isn't the author of the model so let s have some text before that

Reply:
Will edit the input file accordingly.